src/rpa_ri_gpw.F

```diff
@@ -236,27 +216,27 @@ SUBROUTINE rpa_ri_compute_en(qs_env, Erpa, mp2_env, BIb_C, BIb_C_gw, BIb_C_bse_i
                                 mo_coeff_beta, BIb_C_gw_beta, gw_corr_lev_occ_beta, gw_corr_lev_virt_beta)
 
       TYPE(qs_environment_type), POINTER                 :: qs_env
-      REAL(KIND=dp), INTENT(OUT)                         :: Erpa
+      REAL(KIND=dp)                                      :: Erpa
```

Comment:
Why do you remove the intent? (there some more places where that happened)

Reply:
Sorry... I created rpa_gw.F completely new because the GW code has changed a bit due to k-points and moving the routines to rpa_gw.F by myself seemed easier. This is probably also the reason for your points below. I'll try to fix your points...

Reply:
Is there actually a method to automatically assign INTENT(IN), INTENT(OUT) and INTENT(INOUT)?